### DISCUSSION TITLE
Different Python Version/Virtualenv

### DISCUSSION THREAD

**Discussion by ajmorris1** [custom-components/pyscript#247] [Q&A]
Hi All,

Hoping to control the version of python that pyscript uses on the host. 
E.g. I have my tensorflow/opencv environment in a virtualenv, and want it to call some hass actions when it detects classified objects. Can I point pyscript at a different python executable?

Cheers,
Andrew

**Comment by dlashua** [comment]
You cannot. Pyscript runs inside of Home Assistant, therefore, it is bound to whatever version of Python Home Assistant is running.
